PHPCompatibility/Tests/BaseSniffTestCase.php

```diff
@@ -377,4 +377,45 @@ private function gatherIssues($issuesArray)
 
         return $allIssues;
     }
+
+    /**
+     * Run assertions that the fixer output matches the ".fixed" file for a particular sniff.
+     *
+     * @since 10.0.0
+     *
+     * @param string $pathToTestFile   Absolute path to the file to sniff.
+     *                                 Allows for passing __FILE__ from the unit test
+     *                                 file. In that case, the test case file is presumed
+     *                                 to have the same name, but with an `inc` extension.
+     * @param string $targetPhpVersion Value of 'testVersion' to set on PHPCS object.
+     *
+     * @return void
+     */
+    protected function assertFixerOutputMatches($pathToTestFile, $targetPhpVersion = 'none')
+    {
+        $pathToFixedFile = $pathToTestFile . '.fixed';
+
+        $this->assertTrue(file_exists($pathToTestFile), 'Test file exists');
+        $this->assertTrue(file_exists($pathToFixedFile), 'Fixed file exists');
```

Comment:
1. Any particular reason why you are not using [`assertFileExists()`](https://docs.phpunit.de/en/10.5/assertions.html#assertfileexists) here ?
2. The `$message` parameter should reflect the _failure_ message. The text as-is, is the opposite of a failure message. (here and everywhere else)